src/server/array_func.c

```diff
@@ -382,7 +639,7 @@ update_array_indices_remaining_attr(job *parent)
 
 	if (ptbl->tkm_flags & TKMFLG_REVAL_IND_REMAINING) {
 		attribute *premain = &parent->ji_wattr[(int)JOB_ATR_array_indices_remaining];
-		char *pnewstr = cvt_range(parent, JOB_STATE_QUEUED);
+		char *pnewstr = encode_range(&ptbl->tkm_rarr);
```

Comment:
[adding comment on this line as somehow github is not allowing to add comment on exact line in chk_array_doneness()]

@bremanandjk I don't see any changes related to no to save JOB_ATR_array_indices_remaining in db and removal of job_save_db in chk_array_doneness()...